packages/components/src/templates/next/components/internal/CollectionCard/CollectionCard.tsx

```diff
@@ -59,12 +59,12 @@ export const CollectionCard = ({
           </p>
         )}
         {/* TODO: Feature enhancement? Filter by category when clicked */}
-        <p className="prose-label-md mt-3 text-base-content-subtle lg:mt-2">
+        <p className="prose-label-md mt-3 text-base-content-subtle md:mt-2">
           {category}
         </p>
       </div>
       {image && (
-        <div className="relative mt-3 min-h-40 w-full shrink-0 lg:ml-4 lg:mt-0 lg:h-auto lg:w-1/4">
+        <div className="relative mt-3 h-[160px] w-[200px] shrink-0 md:ml-4 md:mt-0">
```

Comment:
double checking because this (`h-[160px] w-[200px]`) changes the styling directly so wanted to check if it's intended. ok with the change in breakpoint from `lg -> md`